src/test_utils.rs

```diff
@@ -13,17 +13,12 @@ pub(crate) struct CompiledContract {
     pub bin: Bytes,
 }
 
-static DEFAULT_L1_PROVIDER_URL: &str = "http://65.21.140.36:8545";
-static DEFAULT_L2_PROVIDER_URL: &str = "http://65.21.140.36:3050";
```

Comment:
moved this to the CI, otherwise we want to default to localhost

Reply:
Yes